### DISCUSSION TITLE
Indestructible ballast flora

### DISCUSSION THREAD

**Discussion by EggsforMe** [FakeFishGames/Barotrauma#15035] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Ballast flora is indestructable, I have been using the plasma cutter for it for 10 minutes

### Reproduction steps

1. Load my campaign (with mods)
2. Get onto ship

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

I unfortunately could not find a way to reproduce the issue, I was able to destroy ballast flora with a plasma cutter with no issues:

https://github.com/user-attachments/assets/4364cca1-e39e-454e-9680-83ccf0a2c455

Are you aware that the flora regenerates health, and the health regeneration becomes faster the larger it grows? Could it be that it had grown so large you can't do enough damage with the plasma cutter (without cutting some branches first)? Could it be that some of the mods you're using modify the ballast flora or the plasma cutter in some way?
